jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/JmeterDsl.java

```diff
@@ -188,7 +188,32 @@ public static DslThreadGroup threadGroup(String name) {
    * @since 0.33
    */
   public static DslSetupThreadGroup setupThreadGroup(ThreadGroupChild... children) {
-    return new DslSetupThreadGroup(Arrays.asList(children));
+    return new DslSetupThreadGroup(null, Arrays.asList(children));
+  }
+
+  /**
+   * This allowing to set a name on the setup thread group.
```

Comment:
For java docs we are following the convention as if the name of the class, method or parameter is prepending the sentence. For example, in this case this same sentence would be written something like: `allows setting the name on the setup thread group.` Also consider following same convention as other similar cases. For instance: `Same as {@link #setupThreadGroup(ThreadGroupChild...)} but allowing to set a name on the thread group.`. You can take transaction for clear example that has similar methods as setupThreadGroup

Reply:
Yes, I tried to understand how it works, now everything is clear